### PR TITLE
Milestone: To Do list: add & remove

### PR DESCRIPTION
**Pull request by kendoriddy**
For this milestone, I achieved the following requirements

- Remove all hardcoded items from the tasks array.
- Create a new JavaScript file for the new functionality.
- Implement a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.
- By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).
- Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1)..
- All changes to the To Do List should be saved in local storage.

### CODE REVIEW

File: src/crud.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
+import dots from '../assets/images/dots.png';
+import deleteIcon from '../assets/images/delete.png';
+import saveIcon from '../assets/images/save.jpeg';
```
Comment (MeqdamAlqudah) [inline src/crud.js:3]: [ ] - Please fix this issue I found in webpack 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @kendoriddy , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Readme file :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 1 above)
Comment (kendoriddy) [reply src/crud.js:3]: Okay thank you. Will do that straight away

Comment (kendoriddy) [reply src/crud.js:3]: Done

File: src/crud.js [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  const id = parseInt(btnId[1], 10);
+  const taskList = document.querySelector(`#tasks-${id}`);
+  const saveEdit = document.querySelector(`.save-${id}`);
+  const deleteEdit = document.querySelector(`.delete-${id}`);
+  const inputId = document.querySelector(`#activity-${id}`);
+  const editBtn = document.getElementById(`${id}`);
+  editBtn.style.display = 'block';
+
+  existingTodos[id].description = inputId.value;
+  localStorage.setItem('todos', JSON.stringify(existingTodos));
```
Comment (victorokoroji) [inline src/crud.js:41]: [OPTIONAL]

- I can see that you used local storage multiple times. I would kindly suggest you write separate functions that will add and get items from localStorage so as to make your code reusable and prevent repeating yourself 😄 .

**Review by victorokoroji — APPROVED** [review]
Hi @kendoriddy, 

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/crud.js (hunk 2 above)
Comment (kendoriddy) [reply src/crud.js:41]: Thank you @vickymarz

**Pull request merged**